__mocks__/got.js

```diff
@@ -0,0 +1,73 @@
+const people = require('./../assets/people.json');
+const films = require('./../assets/films.json');
+const species = require('./../assets/species.json');
+const planets = require('./../assets/planets.json');
+
+
+const got = (url, options) => {
+    let data, isError;
+    if (/^https:\/\/swapi.co\/api\/*/.test(url)) {
+
+        const path = url.replace('https://swapi.co/api/', '').split('/');
+        const pathNum = parseInt(path[1])-1;
+
+        if (isNaN(pathNum) || pathNum > 7) {
+            Promise.resolve({
```

Comment:
Why would `pathNum > 7` be considered a 400 Bad Request? It's still a valid integer, which could still be an id in the future.

If an entry doesn't exist for a given id, that's a 404 Not Found. The request was still valid, there was just no matching resource

Reply:
I am not sure about this one, but I kept pathNum(objectId) restricted less than 7 because I am allowing test only for first 2 people.

Should we put all 87 people in the json file?

Reply:
> Should we put all 87 people in the json file?

Absolutely not. We still only want to replicate a small amount of the data. However you're not thinking of future testing cases where we might bring in data. Why we would want to artificially limit the ids we can pass in this mock? It doesn't make any sense to do.

Reply:
ok got it! Then I will remove that condition